docs/en/observability/monitor-k8s/monitor-k8s-explore-logs-and-metrics.asciidoc

```diff
@@ -0,0 +1,52 @@
+[discrete]
+[[monitor-kubernetes-explore]]
+== Part 3: Explore logs and metrics
+
+Use {kib} to view the metric and log data collected by {agent}.
+Refer to the following sections for more on viewing your data.
+
+* <<monitor-k8s-explore-metrics>>.
+* <<monitor-k8s-explore-logs>>.
+
+[discrete]
+[[monitor-k8s-explore-metrics]]
+=== View performance and health metrics
+
+To view the performance and health metrics collected by {agent}, open
+{kib} and go to **{observability} → Metrics**.
+
+On the **Inventory** page, you can switch between different views to see an
+overview of the containers and pods running on Kubernetes:
+
+[role="screenshot"]
+image::images/metrics-inventory.png[Inventory page that shows Kubernetes pods]
+
+On the **Metrics Explorer** page, you can group and analyze metrics for the
+resources that you are monitoring.
+
+[role="screenshot"]
+image::images/monitor-k8s-metrics-explorer.png[Metrics dashboard that shows CPU usage for Kubernetes pods]
+
+Notice how everywhere you go in {kib}, there is a search bar that allows you to,
```

Comment:
Maybe a screenshot highlighting the searchbar ?? I  am also confused which searchbar is this

Reply:
I think this is just a general comment about how most pages in Kibana have a search function that can help you find things (left in from the previous version of the tutorial). It's probably too general to be really useful in this situation, so I'll just remove it.